### ISSUE TITLE
LWFA example crash

### ISSUE THREAD

**Issue by psychocoderHPC** [ComputationalRadiationPhysics/picongpu#2228]
The LWFA example from the dev branch (d43b1a45409116881c2783d48b778e49ed37f63c + plus alpaka patch https://github.com/ComputationalRadiationPhysics/alpaka/pull/389 ) is crashing after ~1660 steps.

This crash is reproduce able only on CPU with the accelerator `omp2b`.

```
[1,2]<stdout>:Thread 9 "picongpu" received signal SIGSEGV, Segmentation fault.
[1,2]<stdout>:[Switching to Thread 0x7fffd6ffd700 (LWP 26388)]
[1,2]<stdout>:picongpu::currentSolver::Esirkepov<picongpu::particles::shapes::TSC, 3u>::cptCurrent1D<pmacc::cursor::Cursor<pmacc::cursor::TwistAxesAccessor<pmacc::cursor::CT::BufferCursor<pmacc::math::Vector<float, 3, pmacc::math::StandardAccessor, pmacc::math::StandardNavigator, pmacc::math::detail::Vector_components>, pmacc::math::CT::Int<156, 2028, 2147483647> >, pmacc::math::CT::Int<1, 2, 0> >, pmacc::cursor::CT::TwistAxesNavigator<pmacc::math::CT::Int<1, 2, 0>, 3>, pmacc::cursor::CT::BufferCursor<pmacc::math::Vector<float, 3, pmacc::math::StandardAccessor, pmacc::math::StandardNavigator, pmacc::math::detail::Vector_components>, pmacc::math::CT::Int<156, 2028, 2147483647> > >, alpaka::acc::AccCpuOmp2Blocks<std::integral_constant<unsigned long, 3ul>, unsigned int> > (
[1,2]<stdout>:    cellEdgeLength=4.25234222, line=..., cursorJ=..., leaveCell=..., acc=..., 
[1,2]<stdout>:    this=<synthetic pointer>)
[1,2]<stdout>:    at /bigdata/hplsim/scratch/widera/dev/include/picongpu/fields/currentDeposition/Esirkepov/Esirkepov.hpp:217
[1,2]<stdout>:217	                                atomicAdd( &( ( *cursorJ( i, j, k ) ).z() ), accumulated_J, ::alpaka::hierarchy::Threads{} );
[1,2]<stdout>:---Type <return> to continue, or q <return> to quit---Temporary breakpoint 1 at 0x979eac: file /bigdata/hplsim/scratch/widera/dev/thirdParty/alpaka/include/alpaka/atomic/Op.hpp, line 54.
```

**Comment by psychocoderHPC** [comment]
After looking deeper to the issue I found that invalid particles are touched and the particles `localCellIdx` is outside the allowed range.
It could be that the last frame within a supercell is assumed to be full even if not all particles are valid.

**Comment by psychocoderHPC** [comment]
closed with #2229
